examples/sample-wdm-driver/Cargo.toml

```diff
@@ -12,6 +12,9 @@ publish = false
 
 [package.metadata.wdk.driver-model]
 driver-type = "WDM"
+companyname = "Microsoft Corporation"
+copyright = "(C) 2024 Microsoft. All rights reserved."
+productname = "Surface"
```

Comment:
These fields should be using kebab case per toml conventions

Reply:
They also should not be a part of the driver-model metadata section. The `driver-model` section contains things in the driver model tab of visual studio. I think you probably want a separate metadata section for this? What do you think @ayodejiige